multicluster/controllers/multicluster/commonarea/labelidentityimport_controller.go

```diff
@@ -0,0 +1,168 @@
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+
+	multiclusterv1alpha1 "antrea.io/antrea/multicluster/apis/multicluster/v1alpha1"
+	"antrea.io/antrea/multicluster/controllers/multicluster/common"
+)
+
+// LabelIdentityResourceImportReconciler reconciles a LabelIdentity kind of ResourceImport object in the member cluster.
+type LabelIdentityResourceImportReconciler struct {
+	client.Client
+	Scheme                *runtime.Scheme
+	localClusterClient    client.Client
+	localClusterID        string
+	namespace             string
+	remoteCommonArea      RemoteCommonArea
+	installedLabelImports cache.Indexer
+}
+
+func NewLabelIdentityResourceImportReconciler(client client.Client, scheme *runtime.Scheme, localClusterClient client.Client,
+	localClusterID string, namespace string, remoteCommonArea RemoteCommonArea) *LabelIdentityResourceImportReconciler {
+	return &LabelIdentityResourceImportReconciler{
+		Client:                client,
+		Scheme:                scheme,
+		localClusterClient:    localClusterClient,
+		localClusterID:        localClusterID,
+		namespace:             namespace,
+		remoteCommonArea:      remoteCommonArea,
+		installedLabelImports: cache.NewIndexer(resourceImportIndexerKeyFunc, cache.Indexers{}),
+	}
+}
+
+func resourceImportIndexerKeyFunc(obj interface{}) (string, error) {
+	ri := obj.(multiclusterv1alpha1.ResourceImport)
+	return common.NamespacedName(ri.Namespace, ri.Name), nil
+}
+
+func (r *LabelIdentityResourceImportReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+	klog.V(2).InfoS("Reconciling LabelIdentity kind of ResourceImport", "resourceImport", req.NamespacedName)
+	if r.localClusterClient == nil {
+		return ctrl.Result{}, errors.New("localClusterClient has not been initialized properly, no local cluster client")
+	}
+	if r.remoteCommonArea == nil {
+		return ctrl.Result{}, errors.New("remoteCommonArea has not been initialized properly, no remote common area")
+	}
+	var labelIdentityResImport multiclusterv1alpha1.ResourceImport
+	err := r.remoteCommonArea.Get(ctx, req.NamespacedName, &labelIdentityResImport)
+	if err != nil {
+		if !apierrors.IsNotFound(err) {
+			klog.ErrorS(err, "Unable to fetch LabelIdentity kind of ResourceImport", "resourceImport", req.NamespacedName)
+			return ctrl.Result{}, err
+		}
+		labelIdentityResImportObj, exist, _ := r.installedLabelImports.GetByKey(req.NamespacedName.String())
+		if !exist {
+			// Stale controller will clean up any stale LabelIdentities
+			klog.ErrorS(err, "No cached data for LabelIdentity kind of ResourceImport", "resoureImport", req.NamespacedName)
+			return ctrl.Result{}, nil
+		}
+		labelIdentityResImport = labelIdentityResImportObj.(multiclusterv1alpha1.ResourceImport)
+		return r.handleLabelIdentityImpDelete(ctx, &labelIdentityResImport)
+	}
+	return r.handleLabelIdentityImpUpdate(ctx, &labelIdentityResImport)
+}
+
+func (r *LabelIdentityResourceImportReconciler) handleLabelIdentityImpUpdate(ctx context.Context,
```

Comment:
The method should be `handleLabelIdentityImpCreateOrUpdate`

Reply:
Done.